### ISSUE TITLE
Problem building for alpine

### ISSUE THREAD

**Issue by adhawkins** [KenKundert/quantiphy_eval#1]
I'm the maintainer of the Alpine package for quantiphy_eval, and am trying to update it to v0.4. I'm getting the following build error:

```
xcp-alpine-test:~/quantiphy_eval-0.4$ python3 setup.py build
running build
running build_py
creating build
creating build/lib
copying quantiphy_eval.py -> build/lib
xcp-alpine-test:~/quantiphy_eval-0.4$ PYTHONPATH="$PWD/build/lib" pytest
============================= test session starts ==============================
platform linux -- Python 3.8.7, pytest-6.2.1, py-1.10.0, pluggy-0.13.1
rootdir: /home/andy/quantiphy_eval-0.4, configfile: pytest.ini
collected 1 item / 1 error

==================================== ERRORS ====================================
________________________ ERROR collecting test_qeval.py ________________________
test_qeval.py:5: in <module>
    from quantiphy_eval import evaluate, initialize, rm_commas
quantiphy_eval.py:42: in <module>
    class QELexer(Lexer):
quantiphy_eval.py:45: in QELexer
    NAME,
E   NameError: name 'NAME' is not defined
=========================== short test summary info ============================
ERROR test_qeval.py - NameError: name 'NAME' is not defined
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.49s ===============================
```

I have installed the following Alpine packages, which match those listed as dependencies in setup.py:

```
python3 
py3-quantiphy 
py3-inform 
py3-sly 
py3-setuptools 
py3-pytest
```

Can you offer any assistance?

**Comment by adhawkins** [comment]
Now resolved, issue was that I was building sly v0.1, as there were no other tags in that git repo.

**Comment by KenKundert** [comment]
Thanks for the report.  I have now added required versions for all the dependencies.
